Packs/GitHub/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### GitHub IAM
+- Added the **Allow enabling users** parameter.
```

Comment:
not seeing this param in your change.
but you do change other integration params - might want to mention that.